tests/test_graph_types.py

```diff
@@ -1,11 +1,80 @@
 import inspect
 import json
+import sys
+import textwrap
+
+import pytest
 
 from hamilton import graph_types, node
 
 from tests import nodes as test_nodes
 
 
+@pytest.fixture()
+def func_a():
+    """Default function implementation"""
+
+    return textwrap.dedent(
+        """
```

Comment:
Nit -- should be able to have the `def` on the same line as `"""`

Reply:
Triple quotes occupy 3 spaces while indentation is 4 spaces. Having `def` and `"""` hinders readability IMO even though `textwrap.dedent` makes everything safer.

I adjusted the alignment to avoid having the string under the `return` statement though